### DISCUSSION TITLE
I saw an update article that Copilot is now using GPT4 but my Copilot is still running on Codex model.

### DISCUSSION THREAD

**Discussion by mohak2003** [community/community#77881] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am using Student Developer edition of Copilot but its using codex model instead of GPT3.5 Turbo or GPT4 . Why is that ?

**Comment by davecheney** [comment, marked as answer]
If you're referring to the name of the url printed in the output window, please consider that an illustrated example of why putting codenames in public URLs is a bad idea 😉 We've upgraded the model dozens of times over the last two years, the name of the url we expose it on has no relation to the underlying model any more.
